django_celery_beat/migrations/0007_periodictask_origin_key.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('django_celery_beat', '0006_auto_20180210_1226'),
```

Comment:
Other migrations has been added, you'll need to rebase this to be the next one in the list based on what's in the master branch.